docs/api/README.md

```diff
@@ -37,17 +37,15 @@ like to cite it, we would gratefully recommend the following copy:
 
 ## Data licensing
 
-The majority of the data surfaced through this API is more or less just a
-carefully curated mirror of data acquired from various external parties. Such
-data is subject to its original licensing, where applicable.
-
----
+Several datasets surfaced through this API are carefully curated mirrors of data
+acquired from various external parties. Such data is subject to its original
+licensing, where applicable.
```

Comment:
When #283 adds licensing details to all sources, I'd change this to something like "its original licensing, as noted in its source documentation page."

Reply:
Sorry to be pedantic again. Just noting that #283 adds licensing details to `covidcast` sources only. The other Epidata source docs probably still need some work before we can remove this blanket licensing statement.